KEB: configure kyma version per GA ID (#132)
kyma-project/control-plane: components/kyma-environment-broker/README.md

| **APP_DATABASE_PORT** | Defines the database port. | `5432` |
| **APP_DATABASE_NAME** | Defines the database name. | `broker` |
| **APP_DATABASE_SSL** | Specifies the SSL Mode for PostgrSQL. See all the possible values [here](https://www.postgresql.org/docs/9.1/libpq-ssl.html).  | `disable`|
| **APP_KYMA_VERSION** | Specifies the default Kyma version. | None |

> | **APP_VERSION_CONFIG_NAMESPACE** | Defines the Namespace with the ConfigMap that contains Kyma versions for a global account configuration. | None |

Reviewer: ```suggestion
| **APP_VERSION_CONFIG_NAMESPACE** | Defines the Namespace with the ConfigMap that contains Kyma versions for global accounts configuration. | None |
```

> | **APP_VERSION_CONFIG_NAME** | Defines the name of the ConfigMap that contains Kyma versions for a global account configuration. | None |

Reviewer: ```suggestion
| **APP_VERSION_CONFIG_NAME** | Defines the name of the ConfigMap that contains Kyma versions for global accounts configuration. | None |
```

| **APP_DATABASE_PORT** | Defines the database port. | `5432` |
| **APP_DATABASE_NAME** | Defines the database name. | `broker` |
| **APP_DATABASE_SSL** | Specifies the SSL Mode for PostgrSQL. See all the possible values [here](https://www.postgresql.org/docs/9.1/libpq-ssl.html).  | `disable`|
| **APP_KYMA_VERSION** | Specifies the default Kyma version. | None |

> | **APP_ENABLE_ON_DEMAND_VERSION** | Specifies whether a user can specify a Kyma version in a provisioning requst. | `false` |

Reviewer 2: ```suggestion
| **APP_ENABLE_ON_DEMAND_VERSION** | When true allows a user to specify a Kyma version in a provisioning request. | `false` |
```

Author: discussed with our TW, changed to "If set to `true`, a user can specify a Kyma version in a provisioning request."
